.github/workflows/no_old_refs.yml

```diff
@@ -3,28 +3,52 @@
 #
 # Workflow starts when:
 # 1) push on master branch
+# 2) pull request is opened or synchronized or reopened
 #
 #####################################################################################
 name: "Check no closed issue references"
 
 on:
   push:
     branches: [ master ]
+  pull_request:
 
 jobs:
   check_issues:
     runs-on: ubuntu-latest
     steps:
       - name: Download checkstyle
         uses: actions/checkout@v2
-        with:
-          repository: checkstyle/checkstyle
-          ref: master
-          path: checkstyle
-
+      - name: PR linked issues
+        id: links
+        uses: mondeja/pr-linked-issues-action@v2
+        if: github.event_name == 'pull_request'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      #######################################################################
+      # Linked issues, if present, are received in the following format-
+      # 37,38
+      # We convert it into-
```

Comment:
hyphen can be replaced by colon or removed. Same for first line of description

Reply:
Done